### ISSUE TITLE
implementation error

### ISSUE THREAD

**Issue by nevakrien** [Rishit-dagli/GLU#2]
you are not using the dense layer in swiglu 

class SwiGLU(tf.keras.layers.Layer):
    def __init__(self, bias=True, dim=-1, **kwargs):
        super(SwiGLU, self).__init__(**kwargs)
        self.bias = bias
        self.dim = dim
        self.dense = tf.keras.layers.Dense(2, use_bias=bias)

    def call(self, x):
        out, gate = tf.split(x, 2, axis=self.dim) #removed the key word argument for modern tensorflow
        gate = tf.keras.activations.swish(gate)
        x = tf.multiply(out, gate)
        return x
